### PR TITLE
feat: support `zeebe:property`

### PR DESCRIPTION
**Pull request by nikku**
`zeebe:property` works similar to `zeebe:input`:

* can be optional (in this case it is not bound)
* has a string representation (static) but also dynamic (FEEL expression)

-----

Closes https://github.com/camunda/element-templates-json-schema/issues/65

### CODE REVIEW

**Comment by nikku** [comment]
@pinussilvestrus Please checkout if I did integrate things in the right way (first time me working on this project)!

One small feedback:

* I did not see an existing pattern to add bindings (verify these work, but also fail with expected errors), hence I added it for `zeebe:property`. Just feels very natural to test things like this.

File: packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js [hunk 1]
```diff
@@ -182,6 +182,15 @@ describe('validation', function() {
     });
 
 
+    describe('zeebe:property', function() {
+
+      testTemplate('zeebe-property-type');
+
+      testTemplate('invalid-zeebe-property-type', null, true);
```
Comment (pinussilvestrus) [inline packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js:189]:
```suggestion
      testTemplate('invalid-zeebe-property-type');
```

File: packages/zeebe-element-templates-json-schema/resources/schema.json [hunk 2]
```diff
@@ -295,6 +295,7 @@
                           "enum": [
                             "zeebe:input",
                             "zeebe:output",
+                            "zeebe:property",
```
Comment (pinussilvestrus) [inline packages/zeebe-element-templates-json-schema/resources/schema.json:298]: We also need to adjust our [custom error messages](https://github.com/camunda/element-templates-json-schema/blob/master/packages/zeebe-element-templates-json-schema/resources/error-messages.json) (allowed binding types, allowed optional), as they are not part of the raw schema (on purpose, cf. https://github.com/camunda/element-templates-json-schema/issues/56).

File: packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js [hunk 3]
```diff
@@ -182,6 +182,15 @@ describe('validation', function() {
     });
 
 
+    describe('zeebe:property', function() {
```
Comment (pinussilvestrus) [inline packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js:185]: I like this approach 👍 

As you mentioned we are not consistent on this. I think in C7 we have at least some test cases for particular binding types (e.g. `camunda-in-binding`, `camunda-out-binding`), but also not super strict. In C8 we don't have something like this at all, only indirectly in some of the samples. So +1 on this approach.

We can adopt it once we touch it next time.

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
It works ❤️ 

image

Just some minor remarks 👍

File: packages/zeebe-element-templates-json-schema/resources/schema.json (hunk 2 above)
Comment (nikku) [reply packages/zeebe-element-templates-json-schema/resources/schema.json:298]: Fixed via https://github.com/camunda/element-templates-json-schema/pull/69/commits/444ee90326e77142d327e8b01ccc2a7c9d8f9f51 I believe!

**Review by pinussilvestrus — APPROVED** [review]
Looks 👍 Good job!

**Pull request merged**